bot/exts/moderation/incidents.py

```diff
@@ -130,6 +139,39 @@ def has_signals(message: discord.Message) -> bool:
     return ALL_SIGNALS.issubset(own_reactions(message))
 
 
+async def make_message_link_embed(ctx: Context, message_link: str) -> t.Optional[discord.Embed]:
+    """
+    Create an embedded representation of the discord message link contained in the incident report.
+
+    The Embed would contain the following information -->
+        Author: @Jason Terror ♦ (736234578745884682)
+        Channel: Special/#bot-commands (814190307980607493)
+        Content: This is a very important message!
+    """
+    try:
+        message_convert_object = MessageConverter()
+        message = await message_convert_object.convert(ctx, message_link)
+
+    except discord.DiscordException as e:
+        log.exception(f"Failed to make message link embed for '{message_link}', raised exception: {e}")
+
+    else:
+        text = message.content
+        channel = message.channel
+
+        embed = discord.Embed(
+            colour=discord.Colour.gold(),
+            description=(
+                f"**Author:** {format_user(message.author)}\n"
+                f"**Channel:** <#{channel.id}> ({channel.category}/#{channel.name})\n"
+                f"**Content:** {textwrap.shorten(text, 300, placeholder='...')}\n"
```

Comment:
If someone spams without spaces, this will just show "...".

Reply:
Fixed in commit d6872422.